src/history/history.service.ts

```diff
@@ -93,6 +92,16 @@ export class HistoryService {
     return;
   }
 
+  async deleteHistory(user: User): Promise<void> {
```

Comment:
Please add JSDoc annotations for (at least new) service methods

Reply:
added